private/buf/bufwire/bufwire.go

```diff
@@ -202,3 +203,27 @@ func NewImageWriter(
 		fetchWriter,
 	)
 }
+
+// ProtoEncodingWriter is a proto encoding writer.
+type ProtoEncodingWriter interface {
+	// PutMessage writes the message to the value.
+	//
+	// Currently, this only support json format.
```

Comment:
What is "value"? Where is the format specified?

Reply:
Commented on this above - the comment should be updated to be `path`, not `value`.